docker/databricks/00-custom-spark-driver-defaults-alluxio.conf

```diff
@@ -0,0 +1,29 @@
+# Copyright (c) 2022, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+[driver] {
+ "spark.plugins" = "com.nvidia.spark.SQLPlugin"
+ "spark.sql.cache.serializer" = "com.nvidia.spark.ParquetCachedBatchSerializer"
+ "spark.rapids.sql.explain" = "ALL"
+ "spark.rapids.memory.pinnedPool.size" = "2G"
+ "spark.sql.adaptive.enabled" = "false"
+ "spark.databricks.delta.optimizeWrite.enabled" = "false"
+ "spark.sql.optimizer.dynamicPartitionPruning.enabled" = "false"
+ "spark.task.resource.gpu.amount" = "0.25"
+ "spark.sql.files.maxPartitionBytes" = "512m"
+ "spark.rapids.sql.concurrentGpuTasks" = "2"
+ "spark.sql.parquet.int96RebaseModeInRead" = "CORRECTED"
```

Comment:
we shouldn't set int96RebaseModeInRead or datetimeRebaseModeInRead generically, those would be customer specific.